### ISSUE TITLE
A stand-alone command for running a Cheroot server

### ISSUE THREAD

**Issue by scottp-dpaw** [cherrypy/cheroot#86]
* **I'm submitting a ...**
[ ] bug report
[x] feature request
[ ] question about the decisions made in the repository

* **What is the current behavior?**
Applications wishing to support Cheroot have to provide a loader stub to interface with the generic WSGI application.

* **What is the expected behavior?**
Applications can start a Cheroot instance for a WSGI application from the command line without any additional Python code.

* **What is the motivation / use case for changing the behavior?**
My organisation is in the process of migrating several Python WSGI apps to an automated deployment environment (Red Hat OpenShift + S2I). We would be interested in testing Cheroot as a server backend now it has been decoupled from CherryPy, but we do miss being able to spin up an instance from the command line in the manner of Gunicorn and uWSGI.
In short, having a stand-alone application is useful from a testing and deployment standpoint, as it removes the need to ship a dedicated Cheroot loader for a web app. All that is required is a Python WSGI application exposed as a module. 

* **Other information**
The attached pull request contains a proof of concept script (bin/cheroot) which starts a cheroot.wsgi.Server() based on command line arguments, similar in functionality to the standalone gunicorn and uwsgi programs. Given that this is an early stage I would love feedback on the viability of this proposal, the specific command line arguments, etc. (I am holding off on changing the documentation until the design is approved)

**Comment by webknjaz** [comment]
Resolved by #87.

P.S. @scottp-dpaw please add smth like `Resolves #86` to your PRs and/or commits in future, so that GitHub will automatically close related issues upon merge to master :)
